topic-operator/src/main/java/io/strimzi/operator/topic/TopicOperator.java

```diff
@@ -357,16 +372,75 @@ public TopicOperator(Vertx vertx, Kafka kafka,
                          TopicStore topicStore,
                          Labels labels,
                          String namespace,
-                         Config config) {
+                         Config config,
+                         MetricsProvider metrics) {
         this.kafka = kafka;
         this.k8s = k8s;
         this.vertx = vertx;
         this.labels = labels;
         this.topicStore = topicStore;
         this.namespace = namespace;
         this.config = config;
+        this.metrics = metrics;
+
+        initMetrics();
+    }
+
+    public void initMetrics() {
+        Tags metricTags = Tags.of(Tag.of("kind", "KafkaTopic"));
+
+        periodicReconciliationsCounter = metrics.counter(METRICS_PREFIX + "reconciliations.periodical",
```

Comment:
I think "periodic" is better than "periodical"

Reply:
This needs to be same as https://github.com/strimzi/strimzi-kafka-operator/pull/2737/files#diff-94c2cc18347a188f142d49e8bd0c3753R257